hack/gcr-push.sh

```diff
@@ -17,27 +22,31 @@ function latest_release() {
         if [[ "$t" == *"beta"* || "$t" == *"alpha"* || "$t" == *"rc"* ]]; then
             continue
         fi
-        LATEST="$t"
+        HIGHEST="$t"
         break
     done
 }
 
-
-# Always publish for master
 if [ "$BRANCH" == "master" ]; then
-    VERSION="$BRANCH" make all-containers all-push
+    VERSION="$BRANCH"
+elif [ ! -z "$TRAVIS_TAG" ]; then
+    VERSION="$TRAVIS_TAG"
+else
+    # If we're not on master and we're not building a tag, exit early.
+    exit 0
 fi
 
-# Publish when TRAVIS_TAG is defined.
-if [ ! -z "$TRAVIS_TAG" ]; then
-    # Calculate the latest release
-    latest_release
+# Calculate the latest release
+highest_release
 
-    # Default to pre-release
-    TAG_LATEST=false
-    if [[ "$TRAVIS_TAG" == "$LATEST" ]]; then
-        TAG_LATEST=true
-    fi
-
-    VERSION="$TRAVIS_TAG" TAG_LATEST="$TAG_LATEST" make all-containers all-push
+# Assume we're not tagging `latest` by default.
+TAG_LATEST=false
+if [[ "$TRAVIS_TAG" == "$HIGHEST" ]]; then
+    TAG_LATEST=true
 fi
+
+openssl aes-256-cbc -K $encrypted_f58ab4413c21_key -iv $encrypted_f58ab4413c21_iv -in heptio-images-fac92d2303ac.json.enc -out heptio-images-fac92d2303ac.json -d
+gcloud auth activate-service-account --key-file heptio-images-fac92d2303ac.json
+unset GIT_HTTP_USER_AGENT
+
+VERSION="$VERSION" TAG_LATEST="$TAG_LATEST" DOCKER="gcloud docker -- " make all-containers all-push
```

Comment:
isn't `gcloud docker` deprecated? https://cloud.google.com/sdk/gcloud/reference/docker

Reply:
https://github.com/heptio/velero/pull/1800#discussion_r320961780. Travis's build environment is not up to date.